src/parser.rs

```diff
@@ -257,9 +259,40 @@ fn handle_table_macro(
                                 if rust_column_name.is_none() {
                                     rust_column_name = Some(ident);
                                 } else if ident.to_string().eq_ignore_ascii_case("Nullable") {
-                                    column_nullable = true;
+                                    if column_array {
```

Comment:
i would suggest inverting this if and moving the comment above the if, because otherwise we have a branch with nothing in it except a comment

Reply:
it's funny you say this haha, I included this to improve readability.

let's leave it in -- it's easier for me to read